docs/UserGuide.md

```diff
@@ -466,46 +452,82 @@ Examples:
 
 ### Adding a task: `add` `-p`
 
-Adds a task or recurring task to a patient.
+You can add a task or recurring task to a patient with the `add` command.
+
+Format: **`add`** `-p PATIENT_INDEX d/TASK_DESCRIPTION | <DATE TIME> | <INTERVAL TIME_PERIOD>`
 
-Format: `add -p PATIENT_INDEX d/TASK_DESCRIPTION | <DATE TIME> | <INTERVAL TIME_PERIOD>`
+<div markdown="block" class="alert alert-info">
 
-* Adds a task to a patient at the specified `PATIENT_INDEX`.
-* `DATE TIME` and `INTERVAL TIME_PERIOD` must follow the criteria defined in [Task parameters](#task-parameters).
+:information_source: **Notes:**
+* `DATE TIME` and `INTERVAL TIME_PERIOD` must follow the criteria defined in [Task parameters](#parameter-constraints).
+* If `TIME` is omitted, the task will be created with a default time of `0000` hours.
+* `DATE TIME` itself can be omitted as well, this will result in the task being created with a task date and time of 24 hours from the moment of creation.
+* `INTERVAL TIME_PERIOD` is used to specify a recurring task i.e if the Task date passes, it will automatically generate the next Task based on the recurrence.
```

Comment:
```suggestion
* `INTERVAL TIME_PERIOD` specifies a recurring task i.e if the task date passes, it will automatically generate the next task based on the recurrence.
```